templates/scripts/sync_rn_ts_to_all_templates.sh

```diff
@@ -22,5 +22,11 @@ convert() {
   popd
 }
 
+sync_ts_app_code() {
+  rsync -zarv --include="*/" --include="*.ts*" --exclude="*" $1/app $2
```

Comment:
Could we add a step to delete all the target files first? If everything in `app` is always going to be a Typescript file then like we chatted about, we could just `rm -rf app` then `cp -R` the TS `app` directory into place